### PR TITLE
[bitnami/**] Adding the ability to assign to a team member of the containers triage

### PR DESCRIPTION
**Pull request by agomezmoron**
Signed-off-by: Alejandro Gómez <morona@vmware.com>

**Description of the change**

Adding a new workflow to assign new (or reopened) issues/PRs to the Bitnami containers triage team.

**Benefits**

Bitnami team will be notified because team members will be assigned.

**Possible drawbacks**

None detected.

**Applicable issues**

All new issues.

### CODE REVIEW

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  issues:
+    types:
+      - reopened
+      - opened
+  pull_request_target:
+    types:
+      - reopened
+      - opened
+permissions:
+  repository-projects: write
```
Comment (fmulero) [inline .github/workflows/triage.yml:13]: I think this is not needed (there is no projects in this repository)

File: .github/workflows/triage.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  issues: write
+
+# To fix the concurrency when for example more than one label is added
+concurrency:
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
```
Comment (fmulero) [inline .github/workflows/triage.yml:19]: I think this section is not needed. We have only one job and this group is taking into account only `run_id` which is unique for a particular execution, so with this block or without it the workflow can run in parallel if several issues/PR are opened or reopened  at the same time

File: .github/workflows/triage.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+jobs:
+  # For any opened or reopened issue, should be sent into Triage
+  send_to_board:
```
Comment (fmulero) [inline .github/workflows/triage.yml:23]: It doesn't sound clear to me. In this repository there is no project or board

File: .github/workflows/triage.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    if: ${{ github.actor != 'bitnami-bot' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign to a person to work on it
+        uses: pozil/auto-assign-issue@v1.9.0
+        with:
+          numOfAssignee: 1
+          removePreviousAssignees: false
+          teams: "containers-triage"
+          repo-token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
```
Comment (fmulero) [inline .github/workflows/triage.yml:33]: If you use `BITNAMI_BOT_TOKEN`, you can remove completely the `permissions` block to use the default ones

**Review by fmulero — CHANGES_REQUESTED** [review]
Thanks a lot @agomezmoron, Could you have a look to my comments?

File: .github/workflows/triage.yml (hunk 2 above)
Comment (agomezmoron) [reply .github/workflows/triage.yml:19]: Good catch :)

File: .github/workflows/triage.yml (hunk 1 above)
Comment (agomezmoron) [reply .github/workflows/triage.yml:13]: Fully agree @fmulero , good catch!

**Comment by agomezmoron** [comment]
Thanks @fmulero ! Good points! Everything was reviewed. Thanks again!

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**